docs/adr/0008-kommunikasjon-mellom-sbs-og-regel-api.md

```diff
@@ -0,0 +1,48 @@
+# Kommunikasjon mellom applikasjon i sbs og regel-api 
+
+* Status: pending
+* Deciders: Team Dagpenger
+* Date: 2019-10-06 
+
+## Context and Problem Statement
+
+Vår nye dagpengekalkulator er en front-end som skal leve i sbs. Den trenger å snakke med dp-regel-api i fss.
+I fremtiden vil vi også få flere ting som lever i sbs, så vi må finne en fornuftig måte å kommunisere 
+
+## Considered Options
+
+* Sonekryssing 
+* Deploye regel-api i begge soner
+* Bruke kafka som regel-api-endepunkt 
+* … 
+
+## Pros and Cons of the Options 
+
+### Sonekryssing 
+
+#### Cons
```

Comment:
En mulig con er at det er en utgående modell. Håpet/målet er at det erstattes med service mesh i framtida.

Reply:
Hvor langt inne i fremtida er meshen? Kan vi bruke public cloud?

Reply:
@gtcno vet nok mer enn meg, men jeg mistenker det ligger litt for langt inn i framtida. En ting er å få det til å virke internt i clusteret på GCP, men det må jo da også bridges inn i Skya-verden on-prem. Jeg tror ikke det går helt knirkefritt.

Reply:
Jeg tror også at service mesh på tvers av kluster og ikke minst til skya ligger litt for langt unna.